app/lib/page_tracker.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+class PageTracker
+  HISTORY_KEY_PREFIX = "history_for"
+  ORIGIN_PAGES_KEY_PREFIX = "origin_pages_for"
+  TRAINEE_PAGE_URI_REGEX = /trainees\/.+/.freeze
+
+  def initialize(trainee_slug, session, request)
```

Comment:
could we use keyword args? I think ordering is becoming important here

Reply:
Sure